src/Attachment/File.php

```diff
@@ -16,6 +16,7 @@
 use Orchid\Platform\Dashboard;
 use Orchid\Platform\Events\ReplicateFileEvent;
 use Orchid\Platform\Events\UploadFileEvent;
+use phpDocumentor\Reflection\Types\Boolean;
```

Comment:
I don't think we really use it

Reply:
yes you are right ! my vscode missimported it , will remove it .